provision/acc_provision/acc_provision.py

```diff
@@ -185,6 +185,7 @@ def config_default():
                     "end": 65000,
                     "ports_per_node": 3000,
                 },
+                "snat_namespace": "aci-containers-system",
```

Comment:
Per discussion, would prefer to not call this snat_namespace as its more generic, but perhaps we can add a different conf as follow up and deprecate this conf at that time.